qiime2/plugins.py

```diff
@@ -37,6 +37,15 @@ def _init_metadata_(self, ref, factory):
         self._init_data_refs[ref] = factory
         return ref
 
+    def _init_data_collection_(self, ref, records):
```

Comment:
Does it make sense to simplify the signature like this?  You originally suggested

```diff
+    def init_data_collection(self, ref, container_type, *records):
+        if len(records) < 2:
+            raise ValueError('Must provide two or more Artifact inputs.')
+
+        value = self._init_data_collection_(ref, container_type, records)
+        return self._push_record(ref, value, 'init_data_collection')
```

Reply:
Thanks for proposing! I don't think it'll get us what we need, though. I will comment directly on the method defined in the base class.